coralproject/talk
# Update default setting to POST moderation

## What does this PR do?

Sets moderation to POST by default

## How do I test this PR?

From a clean install of Talk, ensure that the initial setting for moderation is POST
